libraries/azurerm_key_vault_secrets.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'azurerm_resource'
+require 'json'
+
+class AzurermKeyVaultSecrets < AzurermPluralResource
+  name 'azurerm_key_vault_secrets'
+  desc 'Verifies settings for a collection of Azure Secrets within to a Vault'
+  example <<-EXAMPLE
+    describe azurerm_key_vault_secrets('vault-101') do
+        it           { should exist }
+    end
+  EXAMPLE
+
+  attr_reader :table
+
+  FilterTable.create
+             .register_column(:ids,          field: :id)
+             .register_column(:attributes,   field: :attributes)
+             .register_column(:contentTypes, field: :contentType) { |table, _value| table[:contentType] || nil }
+             .register_column(:managed,      field: :managed)     { |table, _value| table[:managed] || false }
+             .register_column(:tags,         field: :tags)        { |table, _value| table[:tags] || nil }
```

Comment:
These lines get pretty long so you could just use `_` instead of `_value`, since we don't get anything from it anyway.

On the "or" values, `nil` is already returned when the key doesn't exist, so those could be removed unless it's important to communicate that `nil` is a possible value here. In which case I'd change all of these to use `fetch` (`table.fetch(:contentType, nil)`, etc.) including the one that should return `false` on a miss.